aion/web/channels/question_chronicle.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Aion.QuestionChronicle do
+  @moduledoc """
+    This module saves the time when questions were
+    changed in specific rooms. Thanks to that when a timer
+    fires, it's easy to check if the question should be changed
+    to a new one.
+  """
+  @type t :: %{binary => integer}
+
+  @question_timeout_seconds 10
+  @question_timeout_micro @question_timeout_seconds * 1_000_000
+
+  def question_timeout_micro, do: @question_timeout_micro
+  def question_timeout_milli, do: div @question_timeout_micro, 1000
+
+  def start_link do
+    Agent.start_link(fn -> %{} end, name: __MODULE__)
+  end
+
+  @doc "Lists all question change entries stored by the Agent"
+  @spec list_entries :: __MODULE__.t
+  def list_entries, do: Agent.get(__MODULE__, &(&1))
+
+  @doc "Checks if a question should be changed due to a timeout"
+
```

Comment:
Why empty line here?

Reply:
See my response to the same problem below.